lib/spec/simpleofflinecache.sw-spec.js

```diff
@@ -101,4 +104,36 @@ describe('SimpleOfflineCache instance', function () {
     });
   });
 
+  if (!isChrome) {
+    it('should respect ignoreSearch when matching', function(done) {
+      var cacheName = 'test';
+      var options = {
+        ignoreSearch: true,
+        ignoreMethod: false,
+        ignoreVary: false,
+        prefixMatch: false,
+        cacheName: cacheName
```

Comment:
As we explicitely open a cache, [`cacheName` is simply ignored](https://developer.mozilla.org/en-US/docs/Web/API/Cache/match#Parameters) (actually you are not using it in the implementation). So, given the current implementation, it does not make a lot of sense to include.